<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeapSort</class>
 <widget class="QMainWindow" name="HeapSort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>311</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>HeapSortApp</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
            </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>341</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="array_line">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>inherit</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 6px 6px 6px 6px!important;
                                    font-family: inherit;
                                    font-size: 10pt;
                                    border: 1px solid #ccc;
                                    border-radius: 4px;
                                </string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::VisualMoveStyle</enum>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="info_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 6px 10px 6px 10px!important;
                        font-family: inherit;
                        font-size: 10pt;
                        border: 1px solid #ccc;
                        border-radius: 4px;
                    </string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>160</width>
      <height>136</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>inherit</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
                                    padding: 6px 6px 6px 6px!important;
                                    border: 1px solid #54ad54;
                                    background-color: #54ad54;
                                    color: white;
                                    border-radius: 4px;
                                    font-family: inherit;
                                    font-size: 12pt;
                                    }

                                    QPushButton:hover {
                                    background-color: #288b28;
                                    }

                                    QPushButton:pressed {
                                    background-color: #1B5E20;
                                    }
                                </string>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
       <property name="autoRepeatInterval">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>11</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="sort_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>inherit</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
                                    padding: 6px 6px 6px 6px!important;
                                    border: 1px solid #EF5350;
                                    background-color: #EF5350;
                                    color: white;
                                    border-radius: 4px;
                                    font-family: inherit;
                                    font-size: 12pt;
                                    }

                                    QPushButton:hover {
                                    background-color: #E53935;
                                    }

                                    QPushButton:pressed {
                                    background-color: #B71C1C;
                                    }
                                </string>
       </property>
       <property name="text">
        <string>Сортировать</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>11</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="show_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>inherit</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
                                    background-color: #42A5F5;
                                    color: white;
                                    border: 1px;
                                    border-radius: 4px;
                                    font-family: inherit;
                                    font-size: 12pt;
                                    }

                                    QPushButton:hover {
                                    background-color: #1E88E5;
                                    }

                                    QPushButton:pressed {
                                    background-color: #0D47A1;
                                    }
                                </string>
       </property>
       <property name="text">
        <string>Показать все</string>
       </property>
       <property name="autoRepeatInterval">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="table_widget">
    <property name="enabled">
     <bool>true</bool>
    </property>
<!--    <property name="hidden">-->
<!--     <bool>true</bool>-->
<!--    </property>-->
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>341</width>
      <height>181</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: inherit;
                        font-size: 10pt;
                        border: 1px solid #ccc;
                        border-radius: 4px;
                    </string>
    </property>
    <property name="hidden">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Unsorted Array</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sorted Array</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="select_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>inherit</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                                    padding: 3px 3px 3px 3px!important;
                                    border: 1px solid #bdbdbd;
                                    background-color: #bdbdbd;
                                    color: black;
                                    border-radius: 4px;
                                    font-family: inherit;
                                    font-size: 10pt;
                                    }

                                    QPushButton:hover {
                                    background-color: #9e9e9e;
                                    }

                                    QPushButton:pressed {
                                    background-color: #757575;
                                    }
                                </string>
    </property>
    <property name="hidden">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Выбрать</string>
    </property>
    <property name="autoRepeatInterval">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="del_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>inherit</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                                    padding: 3px 3px 3px 3px!important;
                                    border: 1px solid #bdbdbd;
                                    background-color: #bdbdbd;
                                    color: black;
                                    border-radius: 4px;
                                    font-family: inherit;
                                    font-size: 10pt;
                                    }

                                    QPushButton:hover {
                                    background-color: #9e9e9e;
                                    }

                                    QPushButton:pressed {
                                    background-color: #757575;
                                    }
                                </string>
    </property>
    <property name="hidden">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
    <property name="autoRepeatInterval">
     <number>100</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
